src/com/tekion/cricket/Player.java

```diff
@@ -0,0 +1,126 @@
+package com.tekion.cricket;
+
+public class Player {
+
+    enum State{
```

Comment:
please move this enum to separate file

Reply:
Done sir, I'll push the changes in the week-2 branch.